src/main/java/bio/terra/service/configuration/ConfigEnum.java

```diff
@@ -24,6 +24,8 @@
     FIRESTORE_SNAPSHOT_CACHE_SIZE,
     FIRESTORE_VALIDATE_BATCH_SIZE,
     FIRESTORE_QUERY_BATCH_SIZE,
+    AUTH_CACHE_TIMEOUT_SECONDS,
+    AUTH_CACHE_SIZE,
```

Comment:
You will need to initialize these enum parameters in ConfigurationService.java using getters that you implement in ApplicationConfiguration.java